homeassistant/components/sensor/version.py

```diff
@@ -0,0 +1,21 @@
+from homeassistant.const import MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION
+from homeassistant.helpers.entity import Entity
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+	add_devices([HAversion()])
+
+class HAversion(Entity):
```

Comment:
The class should be named HAVersion to be consistent with the code style I think.